### DISCUSSION TITLE
qt5-default issue

### DISCUSSION THREAD

**Discussion by BrynCurrie** [nrc-cnrc/EGSnrc#1081] [Questions]
I'm setting up a VirtualBox (Version 7.0.10 r158379 (Qt5.15.2)) Linux (22.04.03 LTS (Jammy Jellyfish) virtual machine to run egs_brachy simulations. Following the instructions at https://github.com/nrc-cnrc/EGSnrc/wiki/Install-EGSnrc-on-VirtualBox on issuing the command "sudo apt -y install qt5-default" I get:
Package qt5-default is not available, but is referred to by another package. This may mean that the package is missing, has been obsoleted, or is only available from another source
From my web searches I gather qt5-default has been made obsolete.
Is it appropriate to update this step to:
"sudo apt -y install qtbase5-dev qtchooser qt5-qmake qtbase5-dev-tools"?

**Comment by rtownson** [comment]
Yes that seems appropriate, but I haven't tested myself. Did it work? If so, I will add this to the instructions.

**Reply by BrynCurrie** [reply]
No, I'm working through some dependencies which weren't satisfied on configuring everything (libz was an issue too). Will post a more detailed reply once I've got it to work.

**Comment by crcrewso** [comment]
Pretty sure those were the commands I used to get it installed, though I also default to installing qt creator.
